### ISSUE TITLE
Unexpectedly low number of 1 relationships in as2rel-v4-latest and as2rel-v6-latest

### ISSUE THREAD

**Issue by JustinLoye** [bgpkit/peer-stats#7]
**Problem**
For both as2rel-v4-latest and as2rel-v6-latest, the number of 0 relationships is unexpectedly low.
I am counting 80 times more 0 relationships than 1 relationships.
It does not seem consistent with as2rel-latest, that contains only 4 times more 0s than 1s.

**Investigation**
I assume as2rel-latest give a correct ratio of 0s and 1s since it matches with CAIDA AS relationships dataset (serial 1 and 2, 2024-01-01 snapshot).
The problem is unlikely to come from my parsing because it's also present in the Internet Yellow Pages crawl of as2rel-v4-latest and as2rel-v6-latest.

**Comment by digizeph** [comment]
Thanks for the report. Sounds like a data processing issue. Will investigate soon.

**Comment by digizeph** [comment]
I have patched the data generation pipeline as in PR #8. The data for 2024-01-30 and 2024-01-31 (the latest one) has been updated. The latest files look sane now. Please see the table below:

| data file        | rel 0  | rel 1  | 0-to-1 ratio |
|------------------|--------|--------|--------------|
| [as2rel-latest](https://data.bgpkit.com/as2rel/as2rel-latest.json.bz2)    | 643900 | 169481 | 3.8          |
| [as2rel-v4-latest](https://data.bgpkit.com/as2rel/as2rel-v4-latest.json.bz2) | 586965 | 148369 | 3.9          |
| [as2rel-v6-latest](https://data.bgpkit.com/as2rel/as2rel-v6-latest.json.bz2) | 330713 | 62536  | 5.3          |

Let me know if you see additional issues. I will try to do a historical backfill as soon as this issue's resolution is confirmed.

Thanks!

Commands used for generating this table:
<details>

```
$ oneio https://data.bgpkit.com/as2rel/as2rel-latest.json.bz2 | grep '"rel": 0' | wc -l
  643900
$ oneio https://data.bgpkit.com/as2rel/as2rel-latest.json.bz2 | grep '"rel": 1' | wc -l
  169481
$ oneio https://data.bgpkit.com/as2rel/as2rel-v4-latest.json.bz2 | grep '"rel": 0' | wc -l
  586965
$ oneio https://data.bgpkit.com/as2rel/as2rel-v4-latest.json.bz2 | grep '"rel": 1' | wc -l
  148369
$ oneio https://data.bgpkit.com/as2rel/as2rel-v6-latest.json.bz2 | grep '"rel": 0' | wc -l
  330713
$ oneio https://data.bgpkit.com/as2rel/as2rel-v6-latest.json.bz2 | grep '"rel": 1' | wc -l
  62536
```

Install `oneio` on macOS by `brew install bgpkit/tap/oneio`
</details>

**Comment by JustinLoye** [comment]
Looking good, thank you for the fix.
